tedge/src/cli/disconnect/cli.rs

```diff
@@ -0,0 +1,36 @@
+use crate::{
+    cli::connect::{az::AZURE_CONFIG_FILENAME, c8y::C8Y_CONFIG_FILENAME},
+    command::{BuildCommand, BuildContext, Command},
+};
+
+use super::bridge::DisconnectBridge;
+use structopt::StructOpt;
+
+#[derive(StructOpt, Debug)]
+pub enum TedgeDisconnectBridgeCli {
```

Comment:
I am naming it `TEdge...`. We should stick to one or the other spelling.

Reply:
Lets keep it as is and make a decision later on and convert all occurrences in our code base.